### ISSUE TITLE
[Unblock Legitimate Site] bridge.energi.network

### ISSUE THREAD

**Issue by LeonDolinar** [MetaMask/eth-phishing-detect#14489]
**Domain:**
https://bridge.energi.network/

**Details:** *Some details about the domain (optional)*
Original thread on community forum
https://community.metamask.io/t/deceptive-request-when-bridging-funds/27648

One odd thing is that when I go to https://metamask.github.io/eth-phishing-detect/ it says `This domain is not blocked! No problem here.` but as you can clearly see from the screenshot, that's not the case. Can you please have a look.

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by deshvin** [comment]
@LeonDolinar 
This domain does not appear to be blocked by us or PhishFort. Are you still getting a warning message on your end?

**Comment by deshvin** [comment]
please let us know if you're still getting a phishing warning for this domain by opening a new issue. Thanks!
